packages/office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Basic.Example.tsx

```diff
@@ -89,6 +89,23 @@ export class ComboBoxBasicExample extends React.Component<{}, {
           onClick={ this._basicComboBoxOnClick }
         />
 
+        <ComboBox

```

Comment:
minor suggestion, is it possible to create an example that shows multi-select using a controlled combo box? (just to make sure onChange has enough information.

Reply:
Good point!  Just added it right below the existing single select controlled combobox.